Formula/cacert.rb

```diff
@@ -0,0 +1,87 @@
+class Cacert < Formula
+  desc "Mozilla CA certificate store"
+  homepage "https://curl.se/docs/caextract.html"
+  url "https://curl.se/ca/cacert-2021-07-05.pem"
+  # Ideally we'd have a HTTP mirror, but a GitHub-hosted one is second best.
+  mirror "https://gist.githubusercontent.com/jonchang/b13c60cd50bdd91fd12a034ea53e39d5/raw/e1c430fd9ab59ed34212083537be0da974c26555/cacert-2021-07-05.pem"
+  sha256 "a3b534269c6974631db35f952e8d7c7dbf3d81ab329a232df575c2661de1214a"
+  license "MPL-2.0"
+
+  def install
+    pkgshare.install "cacert-#{version}.pem" => "cacert.pem"
+  end
+
+  def post_install
+    # macOS prior to 10.12.1 doesn't have the Let's Encrypt root, which is commonly used.
+    if OS.mac? && MacOS.version >= :sierra
+      post_install_from_keychain
+    else
+      post_install_from_pem
+    end
+  end
+
+  def post_install_from_keychain
```

Comment:
It's kinda weird to do this here. On new macOS, you install `cacert` for the `post_install`, when you don't even actually use anything it downloads.